cluster/gce/manifests/kube-proxy.manifest

```diff
@@ -1,3 +1,5 @@
+# Please keep kube-proxy configuration in-sync with:
+# cluster/addons/kube-proxy/kube-proxy-ds.yaml
```

Comment:
see lines 1-2 of `cluster/addons/kube-proxy/kube-proxy-ds.yaml`

https://github.com/kubernetes/kubernetes/pull/80566/files#diff-d83ae2fa2959cf05c2e8d4f4a5ac2601R1